apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceToContainAtMostExpectationsTest.kt

```diff
@@ -1,28 +1,30 @@
 package ch.tutteli.atrium.api.fluent.en_GB
 
 import ch.tutteli.atrium.creating.Expect
-
-
-class CharSequenceToContainAtMostExpectationsSpec :
-    ch.tutteli.atrium.specs.integration.CharSequenceToContainAtMostExpectationsSpec(
-        getAtMostTriple(),
-        getAtMostIgnoringCaseTriple(),
-        getNotToContainPair(),
-        getExactlyPair()
-    ) {
+import ch.tutteli.atrium.specs.integration.AbstractCharSequenceToContainAtMostExpectationsTest
+import kotlin.test.Test
+
+class CharSequenceToContainAtMostExpectationsTest : AbstractCharSequenceToContainAtMostExpectationsTest(
+    getAtMostTriple(),
+    getAtMostIgnoringCaseTriple(),
+    getNotToContainPair(),
+    getExactlyPair()
+) {
+    @Test
+    fun trigger_run_gutter() = 1
 
     companion object : CharSequenceToContainSpecBase() {
 
         private fun getAtMostTriple() =
             { what: String, times: String -> "$toContain $what $atMost $times" } to
-                ("$toContain.$atMost.$value/$values" to Companion::toContainAtMost)
+                    ("$toContain.$atMost.$value/$values" to Companion::toContainAtMost)
```

Comment:
I guess this was unintentional, please format the file via intellij (usually CTRL+L)